src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Extentions/HostBuilderExtentions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NLog.Extensions.Logging;
+using Serilog;
+using System;
+
+namespace Exadel.CrazyPrice.Common.Extentions
+{
+    public static class HostBuilderExtentions
+    {
+        public static IHostBuilder UseLogger(this IHostBuilder hostBuilder)
```

Comment:
Probably rename to SetupLogger

Reply:
[Done](https://github.com/yegorl/exadel_discounts_be/pull/6/commits/d20afd5e924712fe206173f9b892ca771e6e6adf)